Python/tdw/py_impact.py

```diff
@@ -341,6 +345,14 @@ def __init__(self, initial_amp: float = 0.5, prevent_distortion: bool = True):
             data = json.loads(Path(resource_filename(__name__, f"py_impact/material_data/{path}.json")).read_text())
             self.material_data.update({mat: data})
 
+        if logging == True:
+            self.mode_properties_log = dict()

```

Comment:
This should be allowed to be an empty dictionary if `logging == False` to prevent errors that could occur if `self.mode_properities_log is None`

In other words, replace these lines with:

`self.mode_properties_log = dict()`